src/java/test/org/apache/zookeeper/server/persistence/UtilTest.java

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
did you mean for this file to be part of the patch?

Reply:
I had to make a bunch of refactorings to move all of these unit tests to the right place. Sorry for polluting the pull request, I can move it to a separate ticket if you want.